src/main/java/com/adobe/commerce/cif/model/cart/Cart.java

```diff
@@ -31,9 +31,6 @@
 
     @ApiModelProperty(value = "The list of the entries in the cart.", required = true)
     protected List<CartEntry> cartEntries;
-
-    @ApiModelProperty(value = "If set, this defines the customer owning this cart. If not set, the cart is an anonymous cart.")
```

Comment:
I'm not sure we should remove this one ... I think it's currently the only way we have to distinguish an anonymous cart from a customer cart ... but I'm not 100% sure we need that.